### ISSUE TITLE
Update marketplace example to version 2

### ISSUE THREAD

**Issue by worming004** [MicrosoftPremier/VstsExtensions#241]
Current marketplace page show an usage example with version 1

https://marketplace.visualstudio.com/items?itemName=mspremier.CreateWorkItem&targetId=470f6f31-5bde-46c7-8ddf-da32fccb15cc&utm_source=vstsproduct&utm_medium=ExtHubManageList

```
# Create or update a work item
- task: CreateWorkItem@1
  displayName: 'Create work item'
  inputs:
    #teamProject: # Optional
    workItemType: 
    title: 
    #assignedTo: # Optional
    [...]
 ```
 
 As now the version to use is version 2, the example given in marketplace do not works.
 
 The marketplace page should show an example usage with version 2

**Comment by ReneSchumacher** [comment]
Hi @worming004,

thanks for pointing this out, I simply missed updating the YAML snippet to `@2`. The rest of the snippet is up to date, though, so it works if you simply replace the `@1` with an `@2`. I will fix the docs right now.

**Comment by worming004** [comment]
Fixed by commit [#887cda673d7a68feecc365699c8de1a6d148f45b](https://github.com/MicrosoftPremier/VstsExtensions/commit/887cda673d7a68feecc365699c8de1a6d148f45b)
